datarepo-clienttests/src/main/java/utils/AuthenticationUtils.java

```diff
@@ -15,7 +18,7 @@
 public final class AuthenticationUtils {
 
   private static volatile GoogleCredentials applicationDefaultCredential;
-  private static volatile ServiceAccountCredentials serviceAccountCredential;
+  private static volatile GoogleCredentials serviceAccountCredential;
```

Comment:
I am curious why these need to be volatile

Reply:
spotbugs gives me an error here if I don't either make these volatile or make the getter synchronized, which seemed worse performance-wise to me.